### ISSUE TITLE
What are the final training loss and accuracy when finetuning?

### ISSUE THREAD

**Issue by oym1994** [openvla/openvla#291]
Hi, thanks for this great job!

May I know the final convergence metrics when finetuning this model? ('train_loss', 'action_accuracy ' and 'l1_loss')
